ci/scripts/github/common.sh

```diff
@@ -56,7 +56,12 @@ export S3_URL="s3://rapids-downloads/ci/mrc"
 export DISPLAY_URL="https://downloads.rapids.ai/ci/mrc"
 export ARTIFACT_ENDPOINT="/pull-request/${PR_NUM}/${GIT_COMMIT}/${NVARCH}/${BUILD_CC}"
 export ARTIFACT_URL="${S3_URL}${ARTIFACT_ENDPOINT}"
-export DISPLAY_ARTIFACT_URL="${DISPLAY_URL}${ARTIFACT_ENDPOINT}"
+
+if [[ "${LOCAL_CI}" == "1" ]]; then
```

Comment:
```suggestion
if [[ ! -z ${LOCAL_CI+x} ]]; then
```
Hint: You can use this to detect if `LOCAL_CI` has been set to anything at all, effectively using it's existence as a flag. ignoring the value.
```bash
if [[ -z ${LOCAL_CI+x} ]]; then
    echo "LOCAL_CI has not been set"
fi
```
```bash
if [[ ! -z ${LOCAL_CI+x} ]]; then
    echo "LOCAL_CI has been set"
fi
```

Reply:
I think in this case this isn't necessary as these script is not directly invoked by users and is instead invoked by other scripts which explicitly set this variable to a value of "1".